src/Extension/Basic/NodeVisitor/TimeNodeVisitor.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the HTML sanitizer project.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HtmlSanitizer\Extension\Basic\NodeVisitor;
+
+use HtmlSanitizer\Extension\Basic\Node\TimeNode;
+use HtmlSanitizer\Model\Cursor;
+use HtmlSanitizer\Node\NodeInterface;
+use HtmlSanitizer\Visitor\AbstractNodeVisitor;
+use HtmlSanitizer\Visitor\HasChildrenNodeVisitorTrait;
+use HtmlSanitizer\Visitor\NamedNodeVisitorInterface;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @final
+ */
+class TimeNodeVisitor extends AbstractNodeVisitor implements NamedNodeVisitorInterface
```

Comment:
You probably want to allow the `datetime` attribute here :) .

Reply:
That's a good call.